ffi/capi/src/provider.rs

```diff
@@ -4,9 +4,9 @@
 
 #[allow(unused_imports)] // feature-specific
 use alloc::boxed::Box;
-use icu_provider::prelude::*;
 #[allow(unused_imports)] // feature-specific
-use icu_provider::MaybeSendSync;
+use icu_provider::any::MaybeSendSync;
```

Comment:
Issue: This trait isn't really `any` related and should probably be in a different module

Reply:
It's only used for bounds in this module, so the only usages outside this module are `AnyPayloadProvider` and one tutorial that uses any.